### DISCUSSION TITLE
Crontab should output nothing

### DISCUSSION THREAD

**Discussion by mathieumd** [sheltermanager/asm3#1359] [General]
Hello,

Since I installed ASM3, I get a daily email from *Cron Daemon*, containing:
> /etc/cron.daily/sheltermanager3:
> config: /etc/asm3.conf

The file `/etc/cron.daily/sheltermanager3` (shipped by Debian package `sheltermanager3` v47) output something on its stderr output, even when it's running fine (I guess). It should only output when something went wrong.

```
$ sudo -i
# cd /usr/lib/sheltermanager3
# python3 cron.py all
config: /etc/asm3.conf
# python3 cron.py all >/dev/null
config: /etc/asm3.conf
# python3 cron.py all 2>/dev/null
# 
```

**Comment by bobintetley** [comment]
agreed, there should be no output as info/errors go to syslog. I've fixed for the next version/package
